_posts/2019-11-28-unique-const-dml-trigger-partitioned-table.md

```diff
@@ -101,7 +98,11 @@ In order to get better understanding about the concept, lets inspect 2 cases. On
 ```
 - Basically, this trigger first checks if there is any record with same SpecificId present in whole table before inserting new ones. Note that, we are joining inserted and target table rather than just selecting from target table. The reason is, there can be `bulk insert` situations in the future. If there is no duplicated SpecificId, then it is safe to insert. Otherwise, just throw an exception. In fact, this trigger decreases performance but since we do not have any other option to ensure uniqueness and there is an index on SpecificId column, performance is not a first priority. There is another interesting point in this trigger definition which is the line:
 
-```
+``` sql
 		SELECT [inserted].[Id] FROM [inserted] WHERE @@ROWCOUNT > 0 AND [inserted].[Id] = scope_identity();
 ```
-- When insert is tested manually through `SSMS` everything works, nothing wrong. But, while related application is trying to insert records to the table by using `EF Core`, the trigger did not work. The reason to this bug is, `EF Core` throws [DbUpdateConcurrencyException](https://github.com/aspnet/EntityFrameworkCore/issues/12064) when entities do not have IdentityColumn thats primary key. And also `EF Core` only cares about latest inserted identity column value, we have to add the [final line of code](https://stackoverflow.com/questions/26896652/) in order it to be worked with `EF Core` as well.
\ No newline at end of file
+- When insert is tested manually through `SSMS` everything works, nothing wrong. But while related application is trying to insert records to the table by using `EF Core`, the trigger did not work. The reason to this bug is, `EF Core` throws [DbUpdateConcurrencyException](https://github.com/aspnet/EntityFrameworkCore/issues/12064) when entities do not have IdentityColumn that is a primary key. And also `EF Core` only cares about latest inserted identity column value, we have to add the [final line of code](https://stackoverflow.com/questions/26896652/) in order it to be worked with `EF Core` as well.
+
+## Conclusion
```

Comment:
Please elaborate. I've sent you a detailed feedback from Slack.

Reply:
new commit is available regarding this.